github-actions/oss-readme-template/src/readme.ts

```diff
@@ -73,6 +77,9 @@ interface SdkHeaderTemplateContext {
   stabilityNotes: string;
 }
 
+const OSS_SDK_USAGE_EXAMPLE_PATH_TEMPLATE =
+  'Check out usage example [here]({{ usageExamplePath }})';
```

Comment:
This part isn't quite what I had in mind, I don't think we necessarily need to enforce consistency on the links to the examples.  We can let the individual SDK READMEs decide how they want to spell out this part, they might want to link to more than one example etc.  So I think we can get rid of this constant (and see my other comments about what I had in mind for the usage example variable below).